### PR TITLE
github login automated

### PR DESCRIPTION
**Pull request by vivek80801**
- [x] github login automated

#3

### CODE REVIEW

File: login-github/index.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+});
+
+let userName = "";
+let password = "";
+
+const ask_user_name = () => {
+  return new Promise((resolve, reject) => {
+    rl.question("What is your user name ? \n ", (answer) => {
+      userName = answer;
+      console.log(`your your user name is ${userName}\n`);
```
Comment (Arsenic-ATG) [inline login-github/index.js:16]: shouldn't it be ?
```js
console.log(`your user name is ${userName}\n`);
```
same is with password

**Review by Arsenic-ATG — CHANGES_REQUESTED** [review]
Just some small changes 🙂

File: login-github/index.js (hunk 1 above)
Comment (vivek80801) [reply login-github/index.js]: Thanks for reviewing.

**Review by Arsenic-ATG — APPROVED** [review]

**Comment by Arsenic-ATG** [comment]
Thanks for contribution @vivek80801 🏅

**Pull request merged**